ozaria/site/components/common/ModalVictory.vue

```diff
@@ -0,0 +1,193 @@
+<script>
+  import BaseModal from './BaseModal'
+  import { mapActions, mapGetters } from 'vuex'
+  import { internationalizeConfig, getNextLevelForLevel, getNextLevelLink } from 'ozaria/site/common/ozariaUtils'
+  import utils from 'core/utils'
+
+  export default Vue.extend({
+    components: {
+      BaseModal
+    },
+    props: {
+      campaignHandle: {
```

Comment:
Are any of these various props mutually exclusive. I.e. The modal only expects a campaignHandle or a currentLevel but not both?
Should that be handled in a console warning explicitly. Otherwise I assume all these props must be fulfilled.

Reply:
It requires campaignHandle and currentLevel at the minimum, and those props are set as required. Do you think it should be validated in created/mounted method as well?

Reply:
Maybe a comment for now?